components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -127,12 +140,14 @@ class InviteMembersStep extends React.PureComponent<Props, State> {
         if (value.indexOf(' ') !== -1 || value.indexOf(',') !== -1) {
             const emails = value.split(/[\s,]+/).filter((email) => email.length).map((email) => ({label: email, value: email, error: !isEmail(email)}));
             const newEmails = [...this.state.emails, ...emails];
-            const {subscriptionStats, cloudUserLimit} = this.props;
+            const {cloudUserLimit} = this.props;
+
+            const showError = this.shouldShowLimitError(newEmails.length);
```

Comment:
just wondering if wouldn't it be clearer to call this constant showLimitError? I think that showError is too generic

Reply:
Hey @pablovelezvidal thanks for the review. I'll take the suggestion. I have changed it to ```showLimitError```